models/issue_comment.go

```diff
@@ -641,5 +643,12 @@ func DeleteComment(comment *Comment) error {
 		}
 	}
 
+	//Delete associated Action
+	if _, err := sess.Delete(&Action{
```

Comment:
Please don't delete actions. Rather check in the dashboard if the comment exists or not and act accordingly.